application/controllers/TodoController.php

```diff
@@ -9,17 +9,19 @@ public function __construct() {
 
 	public function index()
 	{
-		
-		// $this->modules[]      = 'todo';
-        // $this->js_listeners[] = 'Todo.fn.thatsme';
-        // $this->layout('welcome_message');
+		$this->layout('welcome_message');
 	}
 	public function todolist()
 	{
 		$this->modules[]      = 'todo'; //module name
-        $this->js_listeners[] = 'Todo.fn.modal_view()'; 
+		$this->js_listeners[] = 'Todo.fn.modal_view(),Todo.table.tables()'; 
```

Comment:
separate this in 2 lines.
$this->js_listeners[] = 'Todo.fn.modal_view()';
$this->js_listeners[] = 'Todo.table.tables()';